packages/markdown/components/Code/style.scss

```diff
@@ -0,0 +1,128 @@
+@mixin gfmCodeBaseStyles($background: #F6F8FA, $text: inherit) {
```

Comment:
This is just me pulling all the `code`, `pre`, and `kbd`-related styles out of the main GFM stylesheet, and finding-and-replacing various color codes with CSS variable "tokens".